File: C++/Maths/Average calculation.cpp
#include <iostream>
using namespace std;

int main(){
    int num1, num2, num3;
    cout << "Enter the 1st number: ";
    cin >> num1;
    cout << "Enter the 2nd number: ";
    cin >> num2;
    cout << "Enter the 3rd number: ";
    cin >> num3;

    float sum = (num1 + num2 + num3) / 3;
    cout << "Average of numbers = " << sum;
}

File: Java/Maths/Average calculation.java
import java.util.Scanner;
class average {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the 1st number: ");
        int num1 = input.nextInt();
        System.out.print("Enter the 2nd number: ");
        int num2 = input.nextInt();
        System.out.print("Enter the 3rd number: ");
        int num3 = input.nextInt();

        float sum = (num1 + num2 + num3)/3f;
        System.out.println( "Average of numbers = " + sum );
    }
}